tests/basic-tests.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Basic test to ensure PHPUnit is working.
+ *
+ * @package plugin-check
+ */
+
+class Basic_Tests extends WP_UnitTestCase {
+	/**
+	 * Basic test that asserts true.
+	 *
+	 * @test
+	 */
+	public function it_should_assert_true() {
```

Comment:
Nit-pick to keep in mind for the future:
* WordPress core uses the `test_*` method prefix as a convention.
* At some point in Performance Lab we started doing this `it_should_*` convention, and in that plugin it is for now mixed between the two conventions. However I would advise we stick with the core approach.
* Sticking with that convention also means we can avoid the `@test` annotation which is redudant. Usually test methods don't need a doc-block, unless they use a `@dataProvider` or benefit from any extra explanation. Descriptive method names plus inline comments within the test method are usually sufficient to explain what's going on.

This isn't a big issue here since this is obviously just a dummy method, but wanted to point out for future reference. Since there are a few minor other changes to make, maybe let's also rename this to e.g. `test_true()` and remove the doc block?

Reply:
Agreed. I have raise the issue in PL GH about it https://github.com/WordPress/performance/issues/514